pos_default_shipping_date/static/description/index.html

```diff
@@ -0,0 +1,416 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Remove this as will fire a warning during Odoo installation (*)

(*) If you update OCA's maintainer tools it shouldn't be added anymore.